index.html

```diff
@@ -7,6 +7,46 @@
+            <p><strong>Lorem ipsum</strong> dolor sit amet consectetur adipisicing elit. Nobis culpa libero maxime incidunt. Quo quod commodi, maxime vitae corporis magni impedit ea, adipisci quas atque architecto laudantium harum quibusdam ducimus?</p>
+            <p>Adipisci a commodi itaque, ea ipsam delectus ab consectetur sit at maxime optio, aspernatur cum alias totam voluptatibus debitis eveniet rerum. Esse placeat voluptates laborum repellendus nam, tempore porro nisi.</p>
+        </div>
+
+        <div id="blockTwo">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quis praesentium non, numquam sit, provident esse omnis reprehenderit eius tempore facilis in quia eum, autem error atque possimus dicta? Rerum, nostrum.
+            Veniam blanditiis veritatis suscipit, enim cupiditate obcaecati in est esse atque dolores iure eos aperiam neque dicta nihil deleniti tempore itaque. Rerum temporibus atque quo. Tenetur libero enim rerum ipsa.
+            Excepturi corporis animi, facere ut ratione omnis incidunt officiis deleniti tenetur odio ullam totam atque consectetur non, repellendus laboriosam? Dolorem corrupti nemo at necessitatibus aspernatur maiores veniam cum corporis veritatis.
+            Repellat quisquam nihil iusto libero ab beatae vero quas sed quidem. Tenetur dolor nostrum, expedita perferendis corrupti, unde quidem ex mollitia sunt incidunt, dicta deleniti ut. Quos ipsam distinctio quo?
+            In numquam exercitationem, ducimus modi possimus nam itaque temporibus harum vel quidem similique minima cupiditate quod at alias commodi nihil maiores non accusantium explicabo delectus! Obcaecati, sapiente. Quisquam, ducimus vel?
+            Laboriosam distinctio sed, officiis id consequatur hic temporibus placeat non quisquam dolorem est rerum ducimus possimus maiores quidem ipsum, veniam recusandae. Laboriosam ullam quaerat quam nostrum excepturi quis repellendus alias.
+            Quae ad accusantium tempore natus error eveniet quidem dolore, animi nisi illo in facere illum, nobis saepe, quas corrupti vitae at sed necessitatibus magnam ipsam molestiae. Eum eaque reprehenderit facilis.
+            Et consequuntur odit, fuga quod doloribus voluptate enim provident. Explicabo itaque cumque illo voluptate, placeat dignissimos asperiores quaerat aut at, quibusdam cum corrupti. In magni ea assumenda! Aliquam, quasi doloribus.
+            Ratione fugiat totam eos doloremque modi, repudiandae reiciendis deleniti vel atque, eius consequatur explicabo praesentium cupiditate ab est fugit. Error fugit corporis ipsa impedit accusamus atque, excepturi non molestiae quidem?
+            Blanditiis possimus delectus assumenda omnis unde! Natus fuga beatae facilis maxime nostrum minus consectetur? Atque maiores illum dignissimos blanditiis qui ipsa nulla, odio facilis expedita iusto nostrum natus quos itaque.</p>
+        </div>
+    </main>
+    <footer>
+        &copy; 2021 by Dirk
+    </footer>
 </body>
```

Comment:
You could add an additional tab space for all content that belongs into the html, so that the hierarchy between the elements is visually clear.